source/ControllableHero.hx

```diff
@@ -0,0 +1,48 @@
+package;
+
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.FlxG;
+
+class ControllableHero extends FlxSprite
+{
+    public function new()
+        {
+            super();
+            makeGraphic(50, 45, FlxColor.WHITE, false);
```

Comment:
It's good practice to take numbers like these and store them in class-level variables - static properties that can't be changed is usually a good policy for const-type stuff.

Then we can give them names and make the intent of our code clearer - the player's starting_width should be 50, the player's starting_height should be 45

Let's implement this sort of fix here and elsewhere in the project.